src/lelantus.cpp

```diff
@@ -820,7 +837,7 @@ bool GetOutPoint(COutPoint& outPoint, const uint256 &pubCoinValueHash) {
 bool GetReducedOutPoint(COutPoint& outPoint, const uint256 &pubCoinValueHash) {
```

Comment:
This one also needs attention. Nothing is really reduced and even if it was it's not OutPoint. `GetOutPointFromMintTag`? and it's not public coin hash that's passed to the function